kaspresso/src/main/kotlin/com/kaspersky/kaspresso/testcases/core/TestRunner.kt

```diff
@@ -127,11 +137,14 @@ internal class TestRunner<InitData, Data>(
         checkTestInfo(testInfo)
 
         try {
+            val testContext = TestContext(kaspresso, stepsManager, data)
             testRunWatcherInterceptor.onMainSectionStarted(testInfo)
 
+            mainSectionEnricher.beforeMainSectionRun(testInfo, testContext)
```

Comment:
I think it would be safer to send a copy of ```testInfo``` (but not the original instance).

Reply:
Hmm, `TestInfo` already is data class with immutable fields. Do we really need to provide a copy?